cypress/e2e/forms/form-layout/formsLayout.cy.js

```diff
@@ -0,0 +1,9 @@
+describe("Test form", () => {
+    it('test horizontal form', () => {
+        cy.visit("/pages/forms/layouts");
+        cy.get('.form-horizontal #inputEmail3').clear().type('sometext@test.ua');
+        cy.get('.form-horizontal #inputPassword3').clear().type('passwd');
+        cy.get('.checkbox .custom-checkbox').click();
+        cy.get('div:nth-child(3) > div > nb-card > nb-card-body > form > div:nth-child(4) > div > button').click();
```

Comment:
we will learn cy.within command, and this code will look like this: 
```suggestion
         cy.get('.form-horizontal').within(()=>{
            cy.get('#inputEmail3').clear().type('sometext@test.ua');
            cy.get('#inputPassword3').clear().type('passwd');
            cy.get('[type="checkbox"]').check({force:true});
          cy.contains('Sign in').click()
        })
```